tests/benchmarks/tstrfloats_bench.nim

```diff
@@ -0,0 +1,44 @@
+#[
+on OSX:
+nim r -d:danger -d:numIter:100_000_00 tests/benchmarks/tstrfloats_bench.nim
+("toStringSprintf", "genFloatCast", 11.956240000000001)
+("toStringSprintf", "genFloatConf", 1.581176000000001)
+("toStringDragonbox", "genFloatCast", 0.1652149999999999)
+("toStringDragonbox", "genFloatConf", 0.15221700000000027)
+]#
+
+import std/[times, strfloats]
+
+const numIter {.intdefine.} = 10
```

Comment:
so that it runs fast in CI, but users can run a more meaningful benchmark with larger `numIter` (as explained in benchmarks/readme.md)

Reply:
added a note in tests/benchmarks/readme.md to explain this